store/store.go

```diff
@@ -959,6 +978,58 @@ func (s *Store) Find(search *Search, user *auth.UserState) ([]*snap.Info, error)
 	return snaps, nil
 }
 
+// Sections retrieves the list of available store sections.
+func (s *Store) Sections(user *auth.UserState) ([]string, error) {
+	u := *s.sectionsURI // make a copy, so we can mutate it
+
+	q := u.Query()
+
+	q.Set("confinement", "strict")
```

Comment:
Why is this necessary?  We are asking for store sections.. sections don't have confinement settings? Doesn't feel right.